linkerd/app/src/env.rs

```diff
@@ -0,0 +1,1014 @@
+use crate::core::{
```

Comment:
This module is just the old env parsing code moved around, right?

Reply:
yep, though it builds the new structures instead of the big Config